language/use-cases/prod-catalog-enrichment/genai_prod_catalog_enrichment.ipynb

```diff
@@ -0,0 +1,2071 @@
+{
```

Comment:
<p>Why is this in a screenshot instead of just showing the data after it's extracted later?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/378/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>The data size extracted is too big , so for ease for demonstration have only kept the screen grabs of relevant sections </p>